app/config/collections2.php

```diff
@@ -695,6 +695,17 @@
                 'array' => true,
                 'filters' => ['json'],
             ],
+            [
+                '$id' => 'deleted',
```

Comment:
We need to create a compound index for `deleted` and `email` since we'll be searching against both of those columns. That way, we get much better query performance. 

Normally, the order of orders matters in compound indexes (example: `['ASC', 'ASC']` is a different index from `['ASC','DESC']`), but since one column is a boolean, I don't expect a large impact to performance (small impact at worst, none at best).

Reply:
Added index 😇